R/process_data.R

```diff
@@ -45,6 +45,11 @@ process_pacta_results <- function(data, start_year, end_year, time_horizon,
       time_horizon = time_horizon,
       log_path = log_path
     ) %>%
+    remove_sectors_with_missing_production(
```

Comment:
just to make sure, it makes more sense to put this removal after remove_companies_with_missing_exposures than before?

Reply:
I would say so. Missing exposures is a more general case, this just cleans up one edge case that may slip through